cmake/Modules/Packaging.cmake

```diff
@@ -0,0 +1,44 @@
+set(CPACK_PACKAGE_NAME "valkey")
+set(CPACK_PACKAGE_VERSION_MAJOR 8)
+set(CPACK_PACKAGE_VERSION_MINOR 1)
+set(CPACK_PACKAGE_VERSION_PATCH 0)
```

Comment:
1. this is duplicated in `version.h`. can we single-source the version numbers in this file instead?
2. the version number for the unstable branch is 255.255.255. Does it make sense to instruct CPACK to not encode the versions in the packaging file name in this case?

Reply:
We have 2 options here:
1. change version.h into version.h.in and then call `configure_file` on it to replace the placeholders with the actual version number
2. Keep things as is and parse these values from `VALKEY_VERSION` from `version.h`

For Debian systems, this is the standard way: include the version as part of the `.deb` file.

For example, if you are running ubuntu/debian/mint etc you can run:

```
dpkg -l
...
ii  irqbalance                                       1.9.3-2ubuntu5                            amd64        Daemon to ba>
ii  iso-codes                                        4.16.0-1                                  all          ISO language>
ii  isort                                            5.6.4-1                                   all          utility for >
ii  iucode-tool                                      2.3.1-3build1                             amd64        Intel proces>
ii  java-common                                      0.75+exp1                                 all          Base package>
ii  javascript-common                                11+nmu1                                   all          Base support>
ii  jq                                               1.7.1-3build1                             amd64        lightweight >
ii  kbd                                              2.6.4-2ubuntu2                            amd64        Linux consol>
```

You will notice that all packages include the version number.